components/employmentOutlook/FavorabilityDialog.js

```diff
@@ -0,0 +1,152 @@
+import { makeStyles } from '@material-ui/styles';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import Divider from '@material-ui/core/Divider';
+import NextLink from 'next/link';
+import PropTypes from 'prop-types';
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+
+import { DialogContent, DialogTitle, DialogActions } from '../DialogComponents';
+
+const useStyles = makeStyles(theme => ({
+  title: {
+    textAlign: 'center',
+    verticalAlign: 'center',
+    paddingTop: theme.spacing(6),
+    paddingBottom: theme.spacing(4),
+  },
+  emphasis: {
+    fontWeight: 'bold',
+    textDecorationLine: 'underline',
+  },
+  summary: {
+    marginTop: theme.spacing(4),
+    marginBottom: theme.spacing(4),
+  },
+  description: {
+    marginTop: theme.spacing(4),
+    marginBottom: theme.spacing(4),
+  },
+  button: {
+    border: 'none',
+    padding: 0,
+    textDecoration: 'underline',
+  },
+  footer: {
+    justifyContent: 'center',
+    padding: theme.spacing(2, 0, 2, 0),
+  },
+}));
+
+const FAVORABILITY_TYPE = [
+  {
+    value: 'Very Unfavorable',
+    color: '#f96861',
+    growth: 'Declining',
+    size: 'Small',
+    description: `The outlook for this job isn't so good. You may need to expand your search to other counties or switch to another occupation. Right now we don't have any info on the platform about this, so we would need to add activities and figure out how to play this out.`,
+  },
+  {
+    value: 'Unfavorable',
+    color: '#fea830',
+    growth: 'Slow',
+    size: 'Medium',
+    description: `The outlook for this job isn't so good. You may need to expand your search to other counties or switch to another occupation. Right now we don't have any info on the platform about this, so we would need to add activities and figure out how to play this out.`,
+  },
+  {
+    value: 'Favorable',
+    color: '#244cd2',
+    growth: 'Fast',
+    size: 'Medium',
+    description: `The future looks good for this job! The New Jersey Career Network can help you follow the steps you need to take to get employed. 
+    Start by using our employment log to keep track of your day-to-day job search activities. Tracking your search is one of the keys to success. It will also help us learn more about what you're doing so we can recommend the right actions.`,
+  },
+  {
+    value: 'Very Favorable',
+    color: '#6bce7a',
+    growth: 'Moderate',
+    size: 'Large',
+    description: `The future looks good for this job! The New Jersey Career Network can help you follow the steps you need to take to get employed. 
+    Start by using our employment log to keep track of your day-to-day job search activities. Tracking your search is one of the keys to success. It will also help us learn more about what you're doing so we can recommend the right actions.`,
+  },
+];
+
+const getFavorability = favorabilityTypeValue =>
```

Comment:
Might be simpler to just make `FAVORABILITY_TYPES` a map/object rather than an array? I don't imagine them being enumerated over.